circle.yml

```diff
@@ -61,6 +61,11 @@ test:
         parallel: true
     - codecov:
         parallel: true
-    - rm -rf docs/API && make -C docs clean
+    - pyenv local 3.4.3:
+        parallel: true
+    - pip install -r docs-requirements.txt:
+        parallel: true
+    - rm -rf docs/API && make -C docs clean:
+        parallel: true
     - python setup.py docs:
         parallel: true
```

Comment:
> I didn't change anything in the installation process

I've made a note on the specific line.

> So if you can tell me how to restrict steps under test: to certain nodes, I would be very happy...

I cant explain it better than the Circle CI documentation [on the topic](https://circleci.com/docs/1.0/parallelism/).